### DISCUSSION TITLE
57

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#830] [Comments]
http://jojoldu.tistory.com/57

**Comment by jojoldu** [comment]
작성자: 곰길이

정리가 참 잘 되어 있는 것 같습니다. 큰 도움이 되었습니다. 감사합니다.

**Comment by wlehwjdgh** [comment]
안녕하세요.
필요한 경우 
클라이언트로부터 받은 데이터를 db에 INSERT or UPDATE 할때도 캐싱을 사용하는게 
보편적인 방법인가요?
